ql-qlf-plugin/qlf_k6n10f/brams.txt

```diff
@@ -50,24 +50,19 @@ endbram
 
 
 match $__QLF_FACTOR_BRAM36_TDP
-  max dbits 36
-  max abits 15
   min wports 1
   max wports 2
   min rports 1
   max rports 2
-  min efficiency 2
+  min efficiency 1
   shuffle_enable B
   make_transp
   or_next_if_better
 endmatch
 
 match $__QLF_FACTOR_BRAM36_SDP
-  max dbits 36
-  max abits 15
   max wports 1
   max rports 1
-  min efficiency 2
```

Comment:
What's the reason to remove the min efficiency for SDP mode altogether?

Reply:
Some of the really small RAMs have efficiency parameter close to 0 (the majority of TDP36K remain unused) and yosys rounds it to 0. If expected behavior is to infer all RAMs in design and map those to TDP36K blocks we need to remove this rule to allow inferring those small RAMs

Reply:
OK, I will adjust the rules to skip RAMs below 128bits.